ConstantineMaliza/capstone-2
# Query route

#### What does this PR do?
Query route
#### Description of Task to be completed?
Guest can leave query and admin delete 
#### How should this be manually tested?
- clone project
- open editor and go to the terminal run `npm run dev`
- go the postman to check 

1. choose body
2. leave query by required content name, email, and password

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
